themes/jamstackthemes/layouts/theme/single-pricing.html

```diff
@@ -0,0 +1,234 @@
+                                    <li>Setup, provision and deploy yourself</li>
+                                </ul>
+                            </div>
+                            <div class="pricing-footer px-2 pb-3 pt-2">
+                                {{ if $stackbit }}
+                                {{ if $stackbit.createUrl }}
+                                <div class="action-buttons action-buttons-detail">
+                                    <a class="action-button action-button-fill action-button-stackbit" data-action="buy-free" data-context="detail" data-name="{{ .Title }}" href="{{ .Params.github }}/archive/master.zip" >Download .zip</a>
+                                </div>
+                                {{ end }}
+                                {{ end }}
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 col-md-4 mb-2">
+                        <div class="pricing-card pricing-card-website">
+                            <div class="pricing-header p-2">
+                                <h2>Website</h2>
+                                <div class="pricing-cost">
+                                    <span class="pricing-cost-price">$49</span>/
+                                    <span class="pricing-cost-period">one time</span>
+                                </div>
+                            </div>
+                            <div class="pricing-body p-2">
+                                <p class="mb-3">A website based on this theme, with any tech stack you choose. Live in seconds.</p>
+                                <ul>
+                                    <li>Pick NextJS, Gatsby, Hugo, Jekyll, Eleventy, Vuepress or clean HTML</li>
+                                    <li>Select a headless CMS from Sanity, Contentful, DatoCMS or Forestry. Or use the Markdown files only</li>
+                                    <li>Your own Github repo</li>
+                                    <li>Deploy to Netlify, Digital Ocean, Cloudflare, Azure, AWS</li>
+                                </ul>
+                                {{ if $stackbit }}
+
+                            </div>
+                            <div class="pricing-footer px-2 pb-3 pt-2">
+                                {{ if $stackbit.createUrl }}
+                                <div class="action-buttons action-buttons-detail">
+                                    <a class="action-button action-button-fill action-button-stackbit" data-action="buy-once" data-context="detail" data-name="{{ .Title }}" href="{{ $stackbit.createUrl }}&ref=jst&utm_source=jamstackthemes&utm_medium=referral&utm_campaign=custom_themes&utm_content=detail" target="_blank" rel="noopener nofollow">Buy Now</a>
+                                </div>
+                                {{ end }}
+                                {{ end }}
```

Comment:
@JugglerX The indentation of the control structures in this file seems wrong to me. For example, these two `{{ end }}`s that are in the same level but one should be indented in the other. And the opening of the outer one is positioned as if it's in the previous block. I understand the goal of getting the right class names, but I think it makes this file less readable.

I am approving the PR to unblock, but in my opinion, this file deserves some cleanup when someone has some time.